bloom/block_default.go

```diff
@@ -0,0 +1,65 @@
+//go:build purego && parquet.bloom.no_unroll
+
+package bloom
+
+// This file contains direct translation of the algortihms described in the
+// parquet bloom filter spec:
+// https://github.com/apache/parquet-format/blob/master/BloomFilter.md
+//
+// There are no practical reasons to unable the parquet.bloom.no_unroll build
```

Comment:
Maybe "to enable"? Unable is an adjective short for "not able"

Reply:
You are correct 👍